source/ed/data.h

```diff
@@ -52,10 +53,19 @@ namespace ed {
 template <typename T>
 void normalize_uri(std::vector<T*>& vec) {
     std::string prefix = navitia::type::static_data::get()->captionByType(T::type);
+
     for (auto* element : vec) {
-        // Suppression des espaces de l'URI
-        boost::algorithm::replace_all(element->uri, " ", "");
         element->uri = prefix + ":" + element->uri;
+        if (element->uri.find(' ') != std::string::npos) {
```

Comment:
It would be more efficient to add the prefix after the find/replace rather than before